tests/v2/test_waku_swap.nim

```diff
@@ -92,49 +92,50 @@ procSuite "Waku SWAP Accounting":
     await node2.stop()
 
   # TODO Add cheque here
-  asyncTest "Update accounting state after sending cheque":
-    let
-      nodeKey1 = crypto.PrivateKey.random(Secp256k1, rng[])[]
-      node1 = WakuNode.init(nodeKey1, ValidIpAddress.init("0.0.0.0"),
-        Port(60000))
-      nodeKey2 = crypto.PrivateKey.random(Secp256k1, rng[])[]
-      node2 = WakuNode.init(nodeKey2, ValidIpAddress.init("0.0.0.0"),
-        Port(60001))
-      contentTopic = ContentTopic("/waku/2/default-content/proto")
-      message = WakuMessage(payload: "hello world".toBytes(), contentTopic: contentTopic)
-
-    var futures = [newFuture[bool](), newFuture[bool]()]
-
-    # Start nodes and mount protocols
-    await node1.start()
-    node1.mountSwap()
-    node1.mountStore(persistMessages = true)
-    await node2.start()
-    node2.mountSwap()
-    node2.mountStore(persistMessages = true)
-
-    await node2.subscriptions.notify("/waku/2/default-waku/proto", message)
-
-    await sleepAsync(2000.millis)
-
-    node1.wakuStore.setPeer(node2.peerInfo)
-    node1.wakuSwap.setPeer(node2.peerInfo)
-    node2.wakuSwap.setPeer(node1.peerInfo)
-
-    proc handler1(response: HistoryResponse) {.gcsafe, closure.} =
-      futures[0].complete(true)
-    proc handler2(response: HistoryResponse) {.gcsafe, closure.} =
-      futures[1].complete(true)
-
-    # TODO Handshakes - for now we assume implicit, e2e still works for PoC
-    await node1.query(HistoryQuery(contentFilters: @[HistoryContentFilter(contentTopic: contentTopic)]), handler1)
-    await node1.query(HistoryQuery(contentFilters: @[HistoryContentFilter(contentTopic: contentTopic)]), handler2)
-
-    check:
-      (await allFutures(futures).withTimeout(5.seconds)) == true
-      # Accounting table updated with credit and debit, respectively
-      # After sending a cheque the balance is partially adjusted
-      node1.wakuSwap.accounting[node2.peerInfo.peerId] == 1
-      node2.wakuSwap.accounting[node1.peerInfo.peerId] == -1
-    await node1.stop()
-    await node2.stop()
+  # Commenting out this test because cheques are currently not sent after the payment threshold has been reached
```

Comment:
I'd create an issue to re-enable this once soft phase is done and we are in mock phase. Ideally the "soft" and "mock" phase is captured in code, in which case this test would still work with 'mock' setting. Let's do that later though once we got this merged and basic dashboard up, for mock phase

Reply:
Alright. Sounds nice